### PR TITLE
Add initial CODEOWNERS GitHub config file

### PR DESCRIPTION
**Pull request by juliusv**
See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/web/ui @juliusv
+/storage/remote @csmarchbanks @cstyan @bwplotka
```
Comment (tomwilkie) [inline .github/CODEOWNERS:2]:
```suggestion
/storage/remote @csmarchbanks @cstyan @bwplotka @tomwilkie 
```

**Review by roidelapluie — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (roidelapluie) [reply .github/CODEOWNERS:2]: Do you also want to be in MAINTAINERS.md?

Comment (tomwilkie) [reply .github/CODEOWNERS:2]: Sure, happy to.

**Review by csmarchbanks — APPROVED** [review]
:tada:

**Pull request merged**